2-mandatory/2-water-bottle.js

```diff
@@ -15,22 +15,38 @@ You have to implement the missing features according to the specification.
 */
 
 // Here is your starting point:
+
+let emptyBottle = 0;
+let bottleFull = 100;
+let pouringUnits = 10;
+
 let bottle = {
   volume: 0,
   fillUp: function () {
     // calling this function should completely fill your bottle (volume = 100);
+    return this.volume = bottleFull
   },
   pour: function () {
     // calling this function should increase your bottle volume by 10 units;
+    return this.volume += pouringUnits;
   },
   drink: function () {
     // calling this function should decrease your bottle volume by 10 units;
+    return this.volume -= pouringUnits;
   },
   isFull: function () {
     // this function should return true if your bottle is full;
+    if(this.volume === bottleFull) {
+      return true 
+    } else {
+      return false
+    }
   },
   isEmpty: function () {
     // this function should return true if your bottle is empty;
+    if(this.volume === emptyBottle) {
```

Comment:
This is missing a false return when the volume is not empty. I would recommend simplifying this code as mentioned above and this should solve it!